openapi/access-token/bankaxept.yaml

```diff
@@ -16,6 +16,13 @@ paths:
       tags:
         - Client Authorization Service
       summary: Authenticate with the BankAxept ePayment system and receive an Authorization token.
+      parameters:
```

Comment:
Don't we already have `security:  basicAuth: []` defined a couple of lines up? Or did you prefer to be very explicit?

Reply:
The idea was to be very explicit incase anyone to avoid confusing regarding the bCrypt setup. Though I don't have super strong opinions, what do you think is best?

Reply:
I did check and it looks more explicit with your addition, so let's have it for doc purposes. :+1: